test/toUnicode.spec.js

```diff
@@ -6,6 +6,8 @@ const words = [
   [ 'ihr', 'ਹਿਰ' ],
   [ 'imil´o', 'ਮਿਲੵਿੋ' ],
   [ 'iBÎo', 'ਭ੍ਯਿੋ' ],
+  [ 'quohI', 'ਤੋੁਹੀ' ],
+  [ 'qouhI', 'ਤੋੁਹੀ' ],
```

Comment:
Is this the same as the case above?

Reply:
Yes, both `quohI` and `quohI` convert to ਤੋੁਹੀ (`U+0A24` + `U+0A4B` + `U+0A41` + `U+0A39` + `U+0A40`)